contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -279,6 +279,88 @@ NXmpes(NXobject):
             raw(NX_NUMBER):
               doc: |
                 Raw data before calibration.
```

Comment:
We should change the data field in NXdetector to NXdata, and then reference this here (i.e. named "data" in line 275).

Reply:
Done! We are however not changing the base class `NXdetector` where `data` is still of type `NX_NUMBER`.